File: Data Structures & Algorithms/is-anagram/submission-3.java
class Solution {
    public boolean isAnagram(String s, String t) {
        char[] w1 = s.toCharArray();
        char[] w2 = t.toCharArray();

        java.util.Arrays.sort(w1);
        java.util.Arrays.sort(w2);

        
        return java.util.Arrays.equals(w1, w2);
    }
}

File: Data Structures & Algorithms/last-stone-weight/submission-3.py
class Solution:
    def lastStoneWeight(self, stones: List[int]) -> int:
        while len(stones) > 1:
            stones.sort()
            a,b = stones.pop(), stones.pop()

            if a == b:
                continue
            if a > b:
                stones.append(a-b)

        if len(stones) > 0:
            return stones[0]
        else:
            return 0